csrankings-r.csv

```diff
@@ -316,6 +316,7 @@ Ramesh Krishnamurti,Simon Fraser University,https://www.sfu.ca/computing/people/
 Ramesh Kumar Rayudu,Victoria University of Wellington,http://ecs.victoria.ac.nz/Main/RameshRayudu,NOSCHOLARPAGE
 Ramesh Kumar Thakur,KIIT Bhubaneswar,https://cse.kiit.ac.in/profiles/ramesh-kumar-thakur/,W7uKdfQAAAAJ
 Ramesh Loganathan,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/ramesh.loganathan,NOSCHOLARPAGE
+Ramesh Raskar,Massachusetts Institute of Technology,https://www.media.mit.edu/people/raskar/overview,8hpOmVgAAAAJ

```

Comment:
Media Lab, not EECS or CSAIL, needs specific justification for inclusion.

Reply:
He has primarily supervised EECS student as evidenced by this PhD thesis of Connor Henley https://dspace.mit.edu/handle/1721.1/152657 

Doesn’t this meet inclusion criteria?